Refactor

tchalupnik opened the pull request.

tchalupnik committed:
Refactor

tchalupnik commented on nfcclient/nfc_reader/nfc_reader.py:
```diff
@@ -0,0 +1,41 @@
+import logging
+from typing import List
+
+try:
```
WIP
I'm open for suggestions

tchalupnik committed:
Remove init test sh

tchalupnik commented on nfcclient/gpio_client.py:
```diff
@@ -0,0 +1,47 @@
+import asyncio
+import logging
+
+try:
```
WIP
I'm open for suggestions

tchalupnik committed:
Added event loop
Added iter tools to cycle for easier testing on dev

tchalupnik committed:
Fix initial sh nfc module

tchalupnik committed:
Fix remove rest framework

tchalupnik committed:
Added support for tests if env var already existed
Added WEB_PORT env variable
Run web app on port gathered from env variable

kprzekwas reviewed (approved):
I tested manually on raspberry - works.

tchalupnik committed:
Move read_card to different module
Added calendar.py
Added tests for card_reader.py

tchalupnik committed:
Remove calendar

tchalupnik committed:
Remove calendar tests leftovers

tchalupnik committed:
Added request for master card

tchalupnik committed:
Added remote precedent over normal opening

tchalupnik committed:
Added 2 tests

tchalupnik committed:
Added .coveragerc

tchalupnik committed:
Typo

tchalupnik committed:
Added settings.py
Added websockets
Added notifier
Added bootstrap and jquery
Added webservice
Removed flask dependency
Removed dateutil

tchalupnik committed:
Settings changes

tchalupnik committed:
Add Makefile

tchalupnik committed:
Fix settings
Fix tests
